### ISSUE TITLE
URI returned by response is not URL-encoded.

### ISSUE THREAD

**Issue by SawamiWataru** [personium/personium-core#214]
Sample
```
{
  "d": {
    "results": {
      "__metadata": {
        "uri": "https://sample-personium/sample-user-01/__ctl/ExtCell('https://sample-personium/sample-app-00/')",
        "etag": "W/\"1-1524075877220\"",
        "type": "CellCtl.ExtCell"
      },
      "Url": "https://sample-personium/sample-app-00/",
      "__published": "/Date(1524075877220)/",
      "__updated": "/Date(1524075877220)/",
      "_Role": {
        "__deferred": {
          "uri": "https://sample-personium/sample-user-01/__ctl/ExtCell('https://sample-personium/sample-app-00/')/_Role"
        }
      },
      "_Relation": {
        "__deferred": {
          "uri": "https://sample-personium/sample-user-01/__ctl/ExtCell('https://sample-personium/sample-app-00/')/_Relation"
        }
      }
    }
  }
}
```

Correct URI
``` https://sample-personium/sample-user-01/__ctl/ExtCell('https%3A%2F%2Fsample-personium%2Fsample-app-00%2F') ```

**Comment by dixonsiu** [comment]
After the modification, GUI team needs to verify the following sceens.  
1. External Cell List  
![image](https://user-images.githubusercontent.com/29846750/41012560-0cea655a-697d-11e8-8c2e-329ed37a387f.png)  
1. External Role List  
![image](https://user-images.githubusercontent.com/29846750/41012582-2675ab24-697d-11e8-8588-5b62f40885f8.png)

**Comment by SawamiWataru** [comment]
> After the modification, GUI team needs to verify the following sceens.

I confirmed, there was no problem.

**Comment by SawamiWataru** [comment]
Released v1.6.11.
